stock_by_warehouse/tests/test_stock_available_unreserved.py

```diff
@@ -147,12 +147,12 @@ def test01_stock_levels(self):
         self.compare_qty_available_not_res(product_a, 2)
         self.compare_qty_available_not_res(self.template_ab, 2)
 
-        self.assertFalse(product_a.warehouses_stock)
+        self.assertEqual(product_a.warehouses_stock, "{}")
         with Form(product_a) as product:
             product.warehouses_stock_recompute = True
             self.assertEqual(safe_eval(product.warehouses_stock)["warehouse"], 2)
 
-        self.assertFalse(self.template_ab.warehouses_stock)
+        self.assertEqual(self.template_ab.warehouses_stock, "{}")
```

Comment:
It's now possible to declare fields of type JSON, that's maybe more appropriate.

Reply:
Done, solved in https://github.com/Vauxoo/addons-vauxoo/pull/1651